### DISCUSSION TITLE
[cloud_firestore_odm_generator] Respect linting rules on single quotes

### DISCUSSION THREAD

**Discussion by Purus** [firebase/flutterfire#9602] [Feature request] [answered]
When generating the models, generator should respect the linting rules of having single quotes. But currently, the generated code contains double quotes.

Because of this, there are lot of information messages in Android Studios' `Dart Analysis` tab.

``` dart
 return _$FriendsQuery(
      _collection,
      $referenceWithoutCursor: $referenceWithoutCursor.where(
        _$AppUserFieldMap["email"]!,
)
```

<img width="648" alt="image" src="https://user-images.githubusercontent.com/1077735/191789607-c4028c23-f4a7-46d3-a6d6-8986e58ab3a8.png">

**Comment by rrousselGit** [comment, marked as answer]
Single quote vs double quote is a matter of taste. There are rules for both

The true solution is to ignore lints form generated files, or to modify your build.yaml to add custom ignore rules that fit your project.
